README.md

```diff
@@ -10,300 +10,95 @@
-
-exit
-
-	Exit from the command-line.
-
-update <entity> <type> <attributes>
-
-	Update the values of the defined set of attributes, using the following format: name#type=value(|name#type=value)*
-
-append <entity> <type> <attributes>
-
-	Append a new Entity with the defined set of attributes, using the following format: name:type=value(,name:type=value)*
-
-query <entity> <type>
-
-	Get all the information on the selected object.
-
-queryAttr <entity> <type> <attributes>
-
-	Get information on the selected object for the selected attributes.
-
-discover <entity> <type>
-
-	Get all the context providers for a entity and type.
-
-configCb <host> <port> <service> <subservice>
-
-	Config a new host and port for the remote Context Broker.
-
-showConfigCb
-
-	Show the current configuration of the client for the Context Broker.
-
-configIot <host> <port> <service> <subservice>
-
-	Config a new host and port for the remote IoT Agent.
-
-showConfigIot
-
-	Show the current configuration of the client for the IoT Agent.
-
-provision <filename>
-
-	Provision a new device using the Device Provisioning API. The device configuration is
-	read from the file specified in the "filename" parameter.
-
-provisionGroup <template> <data> <type>
-
-	Provision a group of devices with the selected template, taking the information needed to
-	fill the template from a CSV with two columns, DEVICE_ID and DEVICE_NAME. The third parameter, type
-	will be used to replace the DEVICE_TYPE field in the template. All the devices will be provisioned
-	to the same IoT Agent, once the templates have been fulfilled.
-
-listProvisioned
-
-	List all the provisioned devices in an IoT Agent.
-
-removeProvisioned <deviceId>
-
-	Remove the selected provisioned device from the IoT Agent, specified by its Device ID.
-
-addGroup <filename>
-
-	Add a new device group to the specified IoT Agent through the Configuration API. The
-	body is taken from the file specified in the "filename" parameter.
-
-listGroups
-
-	List all the device groups created in the selected IoT Agent for the configured service
-
-removeGroup <apiKey> <resource>
+This project is part of [FIWARE](https://www.fiware.org/). For more information check the FIWARE Catalogue entry for the
+[IoT Agents](https://github.com/Fiware/catalogue/tree/master/iot-agents).
 
-	Remove the device group corresponding to the current configured subservice.
+| :books: [Documentation](https://iotagent-node-lib.rtfd.io) | :mortar_board: [Academy](https://fiware-academy.readthedocs.io/en/latest/iot-agents/idas) | :dart: [Roadmap](https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/roadmap.md) |
+| ---------------------------------------------------------- | ----------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- |
 
-authenticate <host> <port> <user> <password> <service>
+## ToC
```

Comment:
```suggestion
## Content
```

To use the same term than in other repos (such as https://github.com/telefonicaid/fiware-orion)